src/backend/marsha/core/utils/medialive_profiles/medialive-720p.json

```diff
@@ -227,7 +227,8 @@
       "CodecSettings": {
```

Comment:
In your commit message:
- "Since now" > "Up to now"
- "save lot" > "save a lot"
- "loosing quality" > "losing quality"